packages/plugin-compiler/src/test/kotlin/io/realm/compiler/GenerationExtensionTest.kt

```diff
@@ -120,6 +120,16 @@ class GenerationExtensionTest {
         inputs.assertGeneratedIR()
     }
 
+    @Test
+    fun `multiple constructors in model definition`() {
```

Comment:
Since we are not modifying the IR it is maybe overkill to have a test comparing the IR dump. It is annoying to have to update too many files on other changes, so I wouldn't mind removing this.